src/link/Coff.zig

```diff
@@ -1034,6 +1035,10 @@ fn linkWithLLD(self: *Coff, comp: *Compilation) !void {
             try argv.append(p);
         }
 
+        for (self.base.options.lib_dirs) |lib_dir| {
```

Comment:
Is moving this here actually important?

Reply:
Git didnt try hard enough to get a small hunk in this case, maybe I can move it somewhere else so git makes a better hunk.

Reply:
Oh hah, i did accidentally move it. Thanks.